src/components/icon/icon.jsx

```diff
@@ -19,9 +26,10 @@ export default function Icon(props) {
       id={props.id}
       onClick={props.onClick}
       role={props.role}
```

Comment:
The role needs to be `button` if we are going to make the SVG interactive. 

Though, another thing to consider: what if the SVG is wrapped in a button? Like another component called `SVGButton` and it is basically

```
function SVGButton() {
  return (
    <button>
      <svg ...>
        ....
      </svg>
    <button>
  );
}
```

Or we can have the logic / validation in the SVG component itself.

Reply:
Instead of calling it SVGButton we may want to call it IconButton

Reply:
I went with making the `tabindex` based on `role == 'button'`. That seems sufficient. Are there other pros of creating a button wrapper?

Reply:
I think keeping it simpler is better.